product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Tagging AWS resources"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to cloud provider resources created on your own cloud account (Your Cloud Account). Resource tagging allows you to have fine-grained access control and granular cost data at the bucket level. When you add a tag, the following resources are labeled: 
+
+- EC2 instances
+- S3 buckets
+- Load balancers
+- Volumes
+
+You can create and manage tags on a per-project basis in the EDB Postgres® AI Console. The Console then applies the tag to the resources of all regions configured in your account. 
```

Comment:
@amine-khemissi and @ivoryibuaka, I rephrased this part to make it more clear, let me know what you think. 

Cloud Service is the new name that the product team assigned to BigAnimal, and Console is the term to refer to the dashboard. I used Console here as I believe makes it more clear.

Reply:
the phrasing is much better, thank you !